From c1704a34745f8e49373eb2cbe1c83199ab2d0091 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Fri, 9 Dec 2016 18:14:12 -0500
Subject: [PATCH] setfiles: fix build failure after common_bufsiz change

Commit 47cfbf32fd66563f8c4e09ad6cced6abfbe2fad5 ("*: add most of the
required setup_common_bufsiz() calls") switched this tool over to use
the common_bufsiz logic but missed including the header leading to a
build failure when enabled:
selinux/setfiles.c:80:30: error: 'bb_common_bufsiz1' undeclared (first use in this function)

URL: https://bugs.gentoo.org/600106
Reported-by: Jonas Jelten <jj@stusta.net>
Signed-off-by: Mike Frysinger <vapier@gentoo.org>
(cherry picked from commit 1cda8790803a6fb072375c8fa53273333d2d4607)
---
 selinux/setfiles.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/selinux/setfiles.c b/selinux/setfiles.c
index 51a7e63bd741..d8083d4fd580 100644
--- a/selinux/setfiles.c
+++ b/selinux/setfiles.c
@@ -46,6 +46,7 @@
 //usage:     "\n		if it has changed"
 
 #include "libbb.h"
+#include "common_bufsiz.h"
 #if ENABLE_FEATURE_SETFILES_CHECK_OPTION
 #include <sepol/sepol.h>
 #endif
-- 
2.11.0.rc2

